include/libtorrent/session_settings.hpp

```diff
@@ -1481,15 +1495,6 @@ namespace libtorrent
 		// see any traffic from
 		bool ignore_dark_internet;
 
-		// the number of seconds a DHT node is banned if it exceeds the rate
-		// limit. The rate limit is averaged over 10 seconds to allow for bursts
-		// above the limit.
-		int block_timeout;
-
-		// the max number of packets per second a DHT node is allowed to send
-		// without getting banned.
-		int block_ratelimit;
-
```

Comment:
is there a good reason to move these? In general I think it's better to add fields to the end to preserve binary compatibility as much as possible. master isn't stable, so it's not that big of a deal